src/libs/ReportUtils.ts

```diff
@@ -3660,6 +3660,20 @@ function addDomainToShortMention(mention: string): string | undefined {
     return undefined;
 }
 
+/**
+ * Replaces all valid short mention found in a text to a full mention
+ */
```

Comment:
Let's also provide examples of how short-mention strings and long mention strings look in this comment

Reply:
Added an example